packages/xo-web/src/xo-app/logs/index.js

```diff
@@ -119,31 +121,40 @@ const UNHEALTHY_VDI_CHAIN_ERROR = 'unhealthy VDI chain'
 const UNHEALTHY_VDI_CHAIN_LINK =
   'https://xen-orchestra.com/docs/backup_troubleshooting.html#vdi-chain-protection'
 
+const filterOptionRenderer = ({ label, value }) =>
+  _(label, message => <option value={value}>{message}</option>)
```

Comment:
Shouldn't this return a regular node (not an `option`)? `Select` handles the rest.

Reply:
select options not support the regular node

Reply:
But `Select` (from `form/select`) does. So `({ label }) => _(label)` should work.